src/app/fdctl/config/default.toml

```diff
@@ -198,10 +198,24 @@ dynamic_port_range = "8900-9000"
     # with the `--rpc-pubsub-enable-vote-subscription` argument.
```

Comment:
Don't see any updates but can we comment `expected_shred_version` a bit better, of what the default value is, what 0 does, etc in here?